src/main/frontend/handler/global_config.cljs

```diff
@@ -45,10 +45,12 @@
 (defn restore-global-config!
   "Sets global config state from config file"
   []
-  (let [config-dir (global-config-dir)
-        config-path (global-config-path)]
-    (p/let [config-content (fs/read-file config-dir config-path)]
-      (set-global-config-state! config-content))))
+  (if (not-empty @root-dir)
+    (let [config-dir (global-config-dir)
+          config-path (global-config-path)]
+      (p/let [config-content (fs/read-file config-dir config-path)]
+        (set-global-config-state! config-content)))
+    (set-global-config-state! default-content)))
```

Comment:
failback.

Reply:
The `(not-empty @root-dir)` should be able to block non-Electron calls.